src/components/existing-request/existing-request-display.vue

```diff
@@ -641,10 +640,18 @@ export default class ExistingRequestDisplay extends Mixins(
     }
   }
 
-  private showConditionsModal () {
+  protected showConditionsModal () {
     this.setConditionsModalVisible(true)
   }
 
+  /** Called to register the business. */
+  protected async registerYourBusiness (): Promise<void> {
+    // safety check
+    if (!this.isNrApprovedOrConditional) return
+
+    return this.affiliateOrLogin()
+  }
```

Comment:
Again, trivial atm but it might become more complex later.

Also this lets me add in a safety check (so I don't have to assume that the button can only be clicked when the NR is approved or conditional).